### PR TITLE
[VYSH-14] updated README.

### PR DESCRIPTION
**Pull request by bandivyshnavi9**
modified README --jira_link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Test
 
+
```
Comment (mbhargav294) [inline README.md]: Please remove this extra line

Comment (bandivyshnavi9) [reply README.md]: removed.

Comment (mbhargav294) [reply README.md]: Thanks for removing. Seems like you have 3 different commits, can you please squash them together?

Comment (bandivyshnavi9) [reply README.md]: done

Comment (mbhargav294) [reply README.md]: Thanks!
🥇 💯

**Pull request merged**